src/app/pages/events/events.component.scss

```diff
@@ -0,0 +1,326 @@
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.bullet-pt-container {
+    position: absolute;
+    left: 3%;
+    min-width: 65px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+#upcoming-events-heading {
+    text-align: center;
+    text-transform: uppercase;
+    font-size: $font-size-lg;
+    font-weight: 700;
+    letter-spacing: 3px;
+    color: $color-headline;
+}   
+
+app-event-card {
+    // margin-top: 30px;
+    width: 95%;
+    height: 100%;
+    box-shadow: 0px 16px 40px rgba(112, 144, 176, 0.2);
+    border-radius: 20px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+.events-content {
+    height: 500px;
+    display: flex;
+    flex-direction: column;
+    // justify-content: space-evenly;
+    // align-items: center;
+    overflow-y: scroll;
+    // position: relative;
+    padding-top: 0px;
+}
+
+.app-event-card-container {
+    height: 150px;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 30px;
+}
+
+@media screen and (max-width: 930px) {
+    .app-event-card-container {
+        height: 200px;
+        width: 100%;
+        margin-top: 20px;
+        margin-bottom: 10px;
+    }
+}
+
+@media screen and (max-width: 700px) {
+    #heading-underline {
+        height: 6px;
+        margin-bottom: 50px;
+    }
+
+    .sig-list-container {
+        flex-direction: row;
+        align-items: center;
+        width: 90vw;
+        height: 45vh;
+        margin-bottom: 50px;
+    }
+
+    .sig-list-left, .sig-list-right {
+        width: 100%;
+        height: 100%;
+        display: block;
+    }
+
+    .sig-list-left {
+        margin-right: 30px;
+    }
+
+    .bullet-pt {
+        display: none;
+    }
+
+    // .bullet-pt-mobile {
+    //     height: 15px;
+    //     width: 15px;
+    //     display: inline-block;
+    //     border-radius: 50%;
+    //     box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    // }
+
+    .design-sig, .iot-sig, .cybersecurity-sig, .swe-sig, .ai-sig, .general-meetings {
+        width: 95%;
+        height: 85px;
+        // left: 50px;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        border-radius: 10px;
+        padding: 10px;
+        text-align: center;
+        margin-bottom: 20px;
+    }
+    .sig-text {
+        font-size: 0.85rem;
+        text-align: center;
+    }
+
+    .design-sig {
+        background-color: $color-light-pink;
+    }
+
+    .iot-sig {
+        background-color: $color-light-green;
+    }
+
+    .cybersecurity-sig {
+        background-color: $color-light-yellow;
+    }
+
+    .swe-sig {
+        background-color: $color-light-purple;
+    }
+
+    .ai-sig {
+        background-color: $color-neon-light-blue;
+    }
+
+    .general-meetings {
+        background-color: $color-blue;
+    }
```

Comment:
You have the `background-color: _____` defined for each of these up above as well, why not combine the CSS tags instead of redeclaring them below